File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profBankInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profBankInfo extends Auditable {
    public BankInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }

     //    public int ? BankNameId { get; set; }
    //    public virtual BankName BankName { get; set; }

    public bankNameId: number;

    public branchName: string;

    public accountNumber: string;

    public swiftNumber: string;

    public status: string;
    
}

File: Architecture.Core/Entities/ProfBankInfo.cs
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
    public class ProfBankInfo : Auditable
    {
        public int BankInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? BankNameId { get; set; }
        public virtual BankName BankName { get; set; }
        public string BranchName { get; set; }
        public string AccountNumber { get; set; }
        public string SwiftNumber { get; set; }      
        public string Status { get; set; }

    }
}
    